### PR TITLE
Fix NEWS items

### PR DESCRIPTION
**Pull request by batpigandme**
* Change "used" to "uses".
* Clarify the column-naming bullet.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -22,12 +22,12 @@
 *   Updated to latest ReadStat from @evanmiller:
 
     * `read_por()` can now read files from SPSS 25 (#412)
-    * `read_por()` used base-10 instead of base-30 for the exponent (#413)
+    * `read_por()` uses base-10 instead of base-30 for the exponent (#413)
```
Comment (hadley) [inline NEWS.md]: This one actually was correct - it's _supposed_ to use base 30 🤷‍♂️

Comment (batpigandme) [reply NEWS.md]: Should we go to the convention of:
_now uses base-30 instead of base-10 for the exponent_?

Comment (hadley) [reply NEWS.md]: Oh yes, that's much clearer

File: NEWS.md [hunk 2]
```diff
@@ -22,12 +22,12 @@
...
+    * `read_por()` now uses base-30 instead of base-10 for the exponent (#413)
     * `read_sas()` can read zero column file (#420)
     * `read_sav()` reads long strings (#381)
     * `read_sav()` has greater memory limit allowing it to read more labels (#418)
     * `read_spss()` reads long variable labels (#422)
-    * `write_sav()` creates incorrect column names when >10k columns (#410)
+    * `write_sav()` no longer creates incorrect column names when >10k columns (#410)
     * `write_sav()` no longer crashes when writing long label names (#395)
   
 # haven 2.0.0R
```
Comment (batpigandme) [inline NEWS.md]: @hadley, is the R here intentional? If no, I figure might as well get rid of it in this PR.

Comment (hadley) [reply NEWS.md]: I
t's not

**Pull request merged**